### PR TITLE
ARTEMIS-4348 Parse connectors on broker.xml for default URI

### PR DESCRIPTION
**Pull request by clebertsuconic**

### CODE REVIEW

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/ActionAbstract.java [hunk 1]
```diff
@@ -113,7 +115,23 @@ public String getBrokerURLInstance(String acceptor) {
...
+
+                  if (InetAddress.getByName(host).isAnyLocalAddress()) {
+                     host = "localhost";
+                  }
+
+                  return new URI(scheme, null, host, port, null, null, null).toString();
+               }
+            }
          } catch (Exception e) {
+            e.printStackTrace();
```
Comment (brusdev) [inline artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/ActionAbstract.java:134]: Why is not the stack trace printed using `getActionContext().out` ?

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/Transfer.java [hunk 2]
```diff
@@ -43,7 +43,7 @@ public class Transfer extends InputAbstract {
    protected String sourceURL = DEFAULT_BROKER_URL;
 
    @Option(name = "--source-acceptor", description = "Acceptor used to build URL towards the broker. Default: 'artemis'.")
-   protected String sourceAcceptor;
+   protected String sourceAcceptor = "artemis";
```
Comment (brusdev) [inline artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/Transfer.java:46]: The `DEFAULT_BROKER_ACCEPTOR` could be reused here:
```suggestion
   protected String sourceAcceptor = DEFAULT_BROKER_ACCEPTOR;
```

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/ActionAbstract.java (hunk 1 above)
Comment (clebertsuconic) [reply artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/ActionAbstract.java:134]: I didn't mean to commit this line.  Will remove it.

Comment (clebertsuconic) [reply artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/ActionAbstract.java:134]: I will actually keep the e.printStackTrace with your suggestion.

**Review by brusdev — APPROVED** [review]

**Pull request merged**